src/components/views/room_settings/AliasSettings.js

```diff
@@ -65,11 +70,44 @@ module.exports = React.createClass({
 
         if (canonicalAliasEvent) {
             state.canonicalAlias = canonicalAliasEvent.getContent().alias;
+            state.initialCanonicalAlias = state.canonicalAlias;
         }
 
         return state;
     },
 
+    getProposedLocalAliases: function() {
```

Comment:
Wouldn't `this.state.domainToAliases[localDomain]` give you the same here?

Reply:
yes, yes it would. not sure what i was smoking; thanks.